account/crud.js

```diff
@@ -27,11 +27,11 @@
 
 var _ = require('lodash');
 var express = require('express');
-var bodyParser = require('body-parser');
 var CryptoJS = require('crypto-js');
 var jwt = require('jsonwebtoken');
 var router = express.Router();
 var validator = require('validator');
+var passport = require('passport');
 
 var config = require('config.js');
```

Comment:
@hotay can you please tell me how does it worked for you? 

isn't should be  `var config = require('../config');` ?

I think you changed it in PR #45 

Same thing at /firebase-client/index.js line no. 3

I am getting error - Module not found if i keep your copy.

Thanks